sapi/litespeed/lscriu.c

```diff
@@ -289,9 +289,9 @@ static void LSCRIU_Wink_Server_is_Ready(void)
         // Not used for native
         return;
     }
-    if (getenv("LSAPI_UNIQE"))
+    if (getenv("LSAPI_UNIQUE"))
```

Comment:
API Break?

Reply:
Yes, better not touch this, especially as we don't maintain this code.